extensions/sql-bindings/src/common/azureFunctionsUtils.ts

```diff
@@ -246,10 +246,11 @@ export async function addNugetReferenceToProjectFile(selectedProjectFile: string
  * Adds the Sql Connection String to the local.settings.json
  * @param connectionString of the SQL Server connection that was chosen by the user
  */
-export async function addConnectionStringToConfig(connectionString: string, projectFile: string): Promise<void> {
+export async function addConnectionStringToConfig(connectionString: string, projectFile: string, settingName?: string): Promise<void> {
 	const settingsFile = await getSettingsFile(projectFile);
+	let connectionStringSettingName = settingName ? settingName : constants.sqlConnectionStringSetting;
```

Comment:
You can also just set the default in the signature `settingName: string = constants.sqlConnectionStringSetting`